### PR TITLE
OSIRIS APPS 95%

### PR DESCRIPTION
**Pull request by femee24**

### CODE REVIEW

**Comment by femee24** [comment]
@pongstr pa check po if you have the time slamat po :D

**Comment by pongstr** [comment]
merge it when it's 100% done?

**Comment by femee24** [comment]
actually its 95% because of the Twitter issue. its still undecided last Friday wether we used Twitter App or just the Web is enough

and it was decided that we will just be using Web Ui for twitter. :) 
so basically its 100%  :D 
don't want to change the commit name anymore hahaha :D 
thanks

File: modules/groups/manifests/osiris.pp [hunk 1]
```diff
@@ -21,57 +21,86 @@
...
+    install_options => ['--no-binaries','--appdir=/Applications'],
+  }
+
+  package { 'sequel-pro':
+    ensure          => installed,
+    provider        => 'brewcask',
+    install_options => ['--no-binaries','--appdir=/Applications'],
+  }
+
+#  package { 'sqldeveloper':
```
Comment (devcsrj) [inline modules/groups/manifests/osiris.pp]: Remove entire entry instead of commenting them out. Everything else, LGTM

**Comment by femee24** [comment]
I commented that out because I'm migrating the file server to other VM 
just finish extracting

**Comment by devcsrj** [comment]
Merged in [master](https://github.com/TORO-IO/boxen/commits/master) as [`7f99d3a`](https://github.com/TORO-IO/boxen/commit/7f99d3a4927f6fe566fe37869a0d03da9bab88af). Thank you!

**Pull request closed without merging**